core/src/main/java/org/apache/carbondata/core/scan/scanner/impl/BlockletFilterScanner.java

```diff
@@ -113,6 +113,10 @@ public boolean isScanRequired(DataRefNode dataBlock) {
         .get(QueryStatisticsConstants.TOTAL_PAGE_SCANNED);
     totalPagesScanned.addCountStatistic(QueryStatisticsConstants.TOTAL_PAGE_SCANNED,
         totalPagesScanned.getCount() + dataBlock.numberOfPages());
+    QueryStatistic totalBlockletStatistic = queryStatisticsModel.getStatisticsTypeAndObjMap()
```

Comment:
`isScanRequired` is used to check whether this block is required to scan, if it is returning false, we should not add the statistics. 
I think the original code is correct, are you facing some problem?

Reply:
I think "total_blocklets" should be the total of all blocklet, valid_blocklets should be the filtered blocklet. Just like "total_pages", the total_pages is all the pages, not filtered pages.

Reply:
ok, then could you change the output header: "valid_blocklets" change to "filtered_bloclklets"

Reply:
The current blocklet statistics include total_blocklets, valid_blocklets, but the page statistics include total_pages, scanned_pages, and valid_pages. Is it possible to add a scanned_blocklets? Finally, total_blocklets is similar to total_pages, valid_blocklets is similar to valid_pages, and scanned_blocklets is similar to scanned_pages.

Reply:
That is fine